pkg/systemd/logDetails.jsx

```diff
@@ -98,26 +98,23 @@
         const problems = problems_client.proxies('org.freedesktop.Problems2.Entry', '/org/freedesktop/Problems2/Entry');
 
         problems.wait(() => {
-            try {
-                service.GetProblems(0, {})
-                        .then(problem_paths => {
-                            const fields = [entry.PROBLEM_DIR, entry.PROBLEM_DUPHASH, entry.PROBLEM_UUID];
-                            let path = null;
-                            problem_paths.some(pth => {
-                                const p = problems[pth];
-                                if (fields.indexOf(p.ID) > 0 || fields.indexOf(p.UUID) || fields.indexOf(p.Duphash)) {
-                                    path = p;
-                                    return true;
-                                } else {
-                                    return false;
-                                }
-                            });
-
-                            this.setState({ entry: entry, loading: false, error: "", problemPath: path, abrtService: service });
+            service.GetProblems(0, {})
+                    .then(problem_paths => {
+                        const fields = [entry.PROBLEM_DIR, entry.PROBLEM_DUPHASH, entry.PROBLEM_UUID];
+                        let path = null;
+                        problem_paths.some(pth => {
+                            const p = problems[pth];
+                            if (p && (fields.indexOf(p.ID) > 0 || fields.indexOf(p.UUID) || fields.indexOf(p.Duphash))) {
+                                path = p;
+                                return true;
+                            } else {
+                                return false;
+                            }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18044-20221215-124452-32c29ab2-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)